src/ci/docker/run.sh

```diff
@@ -33,7 +33,11 @@ ci_dir="`dirname $script_dir`"
 src_dir="`dirname $ci_dir`"
 root_dir="`dirname $src_dir`"
 
-objdir=$root_dir/obj
+if isCI; then
+    objdir=$root_dir/obj
+else
+    objdir=$root_dir/obj/$image
```

Comment:
I believe nobody have written some scripts based on this directory. :@

Reply:
There are a few CI jobs that hardcode this path, but inside the Docker containers it should be still `/checkout/obj`, so that is hopefully fine. For local usage, people will have to update their scripts, which shouldn't be that difficult.